avatars/avatars.js

```diff
@@ -53,1266 +31,92 @@ window.addEventListener('keydown', e => {
-          return bone;
-        }
-      }
-      return null;
-    }).filter(bone => bone);
-    this.allHairBones = allHairBones;
-    this.hairBones = hairBones;
-
-    this.springBoneManager = null;
-    if (options.hair) {
-      const promiseHair = new Promise((resolve, reject) => {
-        if (!object) {
-          object = {};
-        }
-        if (!object.parser) {
-          object.parser = {
-            json: {
-              extensions: {},
-            },
-          };
-        }
-        if (!object.parser.json.extensions) {
-          object.parser.json.extensions = {};
-        }
-        if (!object.parser.json.extensions.VRM) {
-          object.parser.json.extensions.VRM = {
-            secondaryAnimation: {
-              boneGroups: this.hairBones.map(hairBone => {
-                const boneIndices = [];
-                const _recurse = bone => {
-                  boneIndices.push(this.allHairBones.indexOf(bone));
-                  if (bone.children.length > 0) {
-                    _recurse(bone.children[0]);
-                  }
-                };
-                _recurse(hairBone);
-                return {
-                  comment: hairBone.name,
-                  stiffiness: 0.5,
-                  gravityPower: 0.2,
-                  gravityDir: {
-                    x: 0,
-                    y: -1,
-                    z: 0,
-                  },
-                  dragForce: 0.3,
-                  center: -1,
-                  hitRadius: 0.02,
-                  bones: boneIndices,
-                  colliderGroups: [],
-                };
-              }),
-            },
-          };
-          object.parser.getDependency = async (type, nodeIndex) => {
-            if (type === 'node') {
-              return this.allHairBones[nodeIndex];
-            } else {
-              throw new Error('unsupported type');
-            }
-          };
-        }
-        resolve();
-      });
-      promiseHair.then();
-    }
+const loadData = new Promise((resolve, reject) =>{
+  let count = 2
 
-    // NeedsNewFunc: Pre-rotate
-    const preRotations = {};
-    const _ensurePrerotation = k => {
-      const boneName = modelBones[k].name;
-      if (!preRotations[boneName]) {
-        preRotations[boneName] = new THREE.Quaternion();
-      }
-      return preRotations[boneName];
-    };
-    if (flipY) {
-      _ensurePrerotation('Hips').premultiply(new THREE.Quaternion().setFromAxisAngle(new THREE.Vector3(1, 0, 0), -Math.PI / 2));
+  const progress = (err)=>{
+    if (err){
+      console.log(err)
     }
-    if (flipZ) {
-      _ensurePrerotation('Hips').premultiply(new THREE.Quaternion().setFromAxisAngle(new THREE.Vector3(0, 1, 0), Math.PI));
-    }
-    if (flipLeg) {
-      ['LeftLeg', 'RightLeg'].forEach(k => {
-        _ensurePrerotation(k).premultiply(new THREE.Quaternion().setFromAxisAngle(new THREE.Vector3(1, 0, 0), Math.PI / 2));
-      });
+    if (--count===0){
```

Comment:
I don't think this is following `.eslintrc`: https://github.com/webaverse/app/blob/master/.eslintrc.json

Reply:
I transferred this code, it was before me) you will need to go through the entire code and bring it to beauty